### ISSUE TITLE
LOGO FOR bbbike

### ISSUE THREAD

**Issue by worlwide** [eserte/bbbike#2]
Hello there. I design a logo in my free time. I want to design a logo for you. What do you think ? I want to contribute you.

**Comment by eserte** [comment]
BBBike already has a logo: https://github.com/eserte/bbbike/blob/master/images/srtbike_logo.svg
If you like then you can try to improve it...

**Comment by worlwide** [comment]
Of course.Gladly. I will make a presentation as soon as possible. See you

**Comment by worlwide** [comment]
I hope you like it. What do you think ?

The letter b is associated with the word bicycle.

Different color variations are available.
![bbbike](https://user-images.githubusercontent.com/40164336/41285610-64cb77c4-6e45-11e8-836c-dc4e5d985be3.jpg)
